app/models/form/sales/questions/deposit_amount.rb

```diff
@@ -16,11 +16,7 @@ def initialize(id, hsh, subsection, ownershipsch:, optional:)
   end
 
   def derived?(log)
-    log.outright_sale? && log.mortgage_not_used?
-  end
-
-  def selected_answer_option_is_derived?(_log)
-    true
```

Comment:
This was forcing the question to always be treated as applicable (and thus need an answer for the subsection to count as complete) even when it is derived and not routed to. 

Which given the naming of the function is a weird thing that we should look at further - but this should be enough to sort the existing support ticket.